modules/ocf_irc/manifests/nodejs/apt.pp

```diff
@@ -0,0 +1,18 @@
+class ocf_irc::nodejs::apt {
+  package {
+    'apt-transport-https':;
+  }
+
+  # Install a newer version of Node.js for the Slack bridge and web IRC app
+  apt::key { 'nodejs':
+    id     => '9FD3B784BC1C6FC31A8A0A1C1655A0AB68576280',
+    source => 'https://deb.nodesource.com/gpgkey/nodesource.gpg.key';
+  }
```

Comment:
We can probably just backport for all servers? Even if you fix backporting, we only have a jessie (not jessie-backports) distro in our apt repo. 

You could fix backporting and add a new dist, or backport with an equal version number and pin it with puppet on flood (a total hack), or just backport for all the servers. Your call, they all sound fine to me.